specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2018-02-01-preview/containerregistry_build.json

```diff
@@ -0,0 +1,2220 @@
+        },
+        "steps": {
+          "description": "The list of build steps in a build definition.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BuildStepCreateParameters"
+          }
+        },
+        "triggers": {
+          "description": "A collection of custom triggers for this build definition.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BuildTriggerParameters"
+          }
+        },
+        "alias": {
+          "description": "The alternative updatable name for a build definition.",
+          "type": "string"
+        },
+        "status": {
+          "description": "The current status of build definition.",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "BuildDefinitionStatus",
+            "modelAsString": true
+          }
+        },
+        "timeout": {
+          "format": "int32",
+          "description": "Build Time out in seconds.",
+          "type": "integer"
+        },
+        "isBaseImageTriggersEnabled": {
+          "description": "The value of this property indicates whether auto triggers of the build items trigger a build against this definition.",
+          "default": false,
+          "type": "boolean"
+        }
+      }
+    },
+    "SourceRepositoryCreateParameters": {
+      "description": "The properties for creating the source code repository configuration.",
+      "required": [
+        "sourceControlType",
+        "repositoryUrl"
+      ],
+      "type": "object",
+      "properties": {
+        "sourceControlType": {
+          "description": "The type of source control service.",
+          "enum": [
+            "Github",
+            "VisualStudioTeamService"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SourceControlTypes",
+            "modelAsString": true
+          }
+        },
+        "repositoryUrl": {
+          "description": "The type of source control service.",
+          "type": "string"
+        },
+        "sourceControlAuthProperties": {
+          "$ref": "#/definitions/SourceControlAuthInfo",
+          "description": "The authorization properties for accessing the source code repository."
+        },
+        "isCommitTriggerEnabled": {
+          "description": "The value of this property indicates whether the source control commit trigger is enabled or not.",
+          "default": false,
+          "type": "boolean"
+        }
+      }
+    },
+    "BuildStepCreateParameters": {
+      "description": "The base properties for creating any build step.",
+      "required": [
+        "name"
+      ],
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The type of the step.",
+          "enum": [
+            "Docker"
```

Comment:
Are you expecting more values?

Reply:
Yes, may be vulnerability scanning or some other type of building.